tests/backup/backup_share_test.go

```diff
@@ -2393,14 +2399,16 @@ var _ = Describe("{ClusterBackupShareToggle}", func() {
 				}
 				log.InfoD("Waiting for 15 minutes for the next schedule backup to be triggered")
 				time.Sleep(15 * time.Minute)
+				latestScheduleBkpName, err := GetLatestScheduleBackupName(ctx, scheduleName, BackupOrgID)
```

Comment:
The GetLatestScheduleBackupName does not necessarily return to the second backup name. You might want to use GetOrdinalScheduleBackupName instead.

```go
secondSchBackupName, err := GetOrdinalScheduleBackupName(ctx, scheduleName, 2, BackupOrgID)
dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the second schedule backup [%s]", secondSchBackupName))
```

Reply:
Yes yes, here I need to get the latest schedule backup only, not the second one